functions/src/v1/Services/Scheduled/callCarAPI.ts

```diff
@@ -32,9 +32,10 @@ export async function callCarAPI() {
       'Content-Type': 'application/json',
     },
     body: JSON.stringify({date: midnightDate, spaceType: 'car'}),
-    // retry up to 3 times with backoff if 429 status code
+    // retry up to 3 times with backoff if >400 status code
     retries: 3,
-    retryOn: [429],
+    retryOn: async (_attempt, error, response) =>
+      error !== null || (response !== null && response.status >= 400),
```

Comment:
Why make this broader?

I think 429 was specifically for when we get cold start issues

Reply:
This was your change 🤣 
https://github.com/and-murray/wharf-spaces/commit/3fd3c020e0d28ae4dc76f5a71862665c40466e68